Lib/site.py

```diff
@@ -470,11 +471,18 @@ def register_readline():
 
         # Reading the initialization (config) file may not be enough to set a
         # completion key, so we set one first and then read the file
-        if 'libedit' in getattr(readline, '__doc__', ''):
+        if 'libedit' in (getattr(readline, '__doc__', '') or ''):
```

Comment:
Why would `getattr()` fail to return the default value here?

Reply:
The problem here is that `getattr`'s default value is returned only when the attribute does not exist, and in the use case it indeed exists, with value None, triggering the error (NoneType is not iterable).

Please see [bpo-18852](https://bugs.python.org/issue18852) for more details. Personally, I've never had this problem (I don't use Windows), I just thought that importing that small fix could be useful for people using Windows.

Reply:
Right. OK.